specification/imds/data-plane/Microsoft.InstanceMetadataService/stable/2020-10-01/imds.json

```diff
@@ -878,6 +914,136 @@
           "description": "This is the AAD tenantId of the identity of the system assigned managed identity."
         }
       }
+    },
+    "LoadBalancerMetadata": {
+      "type": "object",
+      "description": "This is the response from the Loadbalancer_GetMetadata operation.",
+      "properties": {
+        "publicIpAddresses": {
+          "type": "array",
+          "description": "List of the instance level public or private IP of VM.",
+          "items": {
+            "type": "object",
+            "description": "",
+            "$ref": "#/definitions/PublicIpAddress"
+          }
+        },
+        "inboundRules": {
+          "type": "array",
+          "description": "List of load balancing rules or inbound NAT rules, using which the Load Balancer directs traffic to the VM. Frontend IP addresses and the Private IP addresses listed here belong to the Load Balancer.",
+          "items": {
+            "type": "object",
+            "description": "",
+            "$ref": "#/definitions/LoadBalancerInboundRule"
+          }
+        },
+        "outboundRules": {
+          "type": "array",
+          "description": "List of outbound rules, by which the VM behind Load Balancer sends outbound traffic. Frontend IP addresses and the Private IP addresses listed here belong to the Load Balancer.",
+          "items": {
+            "type": "object",
+            "description": "",
+            "$ref": "#/definitions/LoadBalancerOutboundRule"
+          }
+        },
+        "tunnelInterfaces": {
+          "type": "array",
+          "description": "List of tunnel interfaces, by which the VM behind Gateway Load Balancer sends/receives traffic to/from consumer Loadbalancer or Internet. This field is only available when VM is in the backend pool of a Gateway Load Balancer. Learn more at aka.ms/gatewaylb.",
```

Comment:
Will the tunnelInterfaces field be available from the consumer backend pool - eg. if IMDS is used for a VM behind the consumer NIC/LB, what will they see here?